module-4/backend/week_three.md

```diff
@@ -1,19 +1,66 @@
 ## Week Three - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. At a high level, what is Node?
 
-Note: When you're done, submit a PR. 
+It's a server side runtime environment for javascript. 
 
-1. At a high level, what is Node?
 2. What is Express? What is Express similar to in the Ruby world?
+
+Express is a node web app framework; Express is to JS as Sinatra is to Ruby in terms of provided functionality. 
+
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+// HTTP verb + URL + controller + action
+```app.get('/api/v1/foods', FoodsController.indexFood)
+app.post('/api/v1/foods', FoodsController.createFood)
+app.put('/api/v1/foods/:id', FoodsController.editFood)
+app.delete('/api/v1/foods/:id', FoodsController.deleteFood)```
+
+*if a controller and model objects not created yet, the second argument would be a function(request, response){} to handle the request and response data
+
 4. What do we use Knex for?
+
+It is the ORM for node/js. It allows us to connect to and communicate with a databbase by making queries. 
```

Comment:
i don't know if i'd quite say it's an ORM, but it does connect and communicate with a database